ttnn/cpp/ttnn/operations/pool/downsample/downsample.hpp

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "ttnn/operations/core/core.hpp"
+#include "ttnn/run_operation.hpp"
```

Comment:
I think only `#include ttnn/decoratators.hpp` is needed

Reply:
Done.